regolith/helper.py

```diff
@@ -22,6 +22,7 @@
 from regolith.helpers import u_institutionshelper as u_institutions
 from regolith.helpers import a_todohelper as a_todo
 from regolith.helpers import v_meetingshelper as v_meetings
+from regolith.helpers import g_todohelper as g_todo
```

Comment:
I think this is an adder, isn't it?  So it should be ``a_`` something.   maybe ``a_action``.  But I guess one question is why we need it if we have ``a_todo``?  Can't the meeting scribe run a_todo if an action comes up?

But then we want it on the group meeting (or any meeting I guess) minutes so we can check it next week, right?  But a better way to do that would be to have a method in group meeting helper itself.  Right now they type something like  ``python edit_grpmtg.py 2020-07-12 --actions "(everyone) clear milestones"``.  The idea would be to change that behavior so the program also takes an assignee.  The best might be to change it to ``--action`` (singular) so it only takes one action (this is by far the most common situation) and action takes exactly two args, ``action_assignee`` and ``action_task``, so the above example would be ``python edit_grpmtg.py 2020-07-12 -a everyone "clear milestones"``

Its ok to have this edit_grpmtg in a python file for now, but pls could you make an issue to make this into an updater helper?  But only after these changes are made.

Reply:
@sbillinge, I'm not quite sure what the updater helper refers to.. is it to update actions for a group meeting?